File: Python/Invert Binary Tree/Invert Binary Tree.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, x):
#         self.val = x
#         self.left = None
#         self.right = None


class Solution:
    def invertTree(self, root: TreeNode) -> TreeNode:
        self.invert(root)
        return root

    def invert(self, node: TreeNode):
        if(node == None):
            return
        tmp = node.left
        node.left = node.right
        node.right = tmp
        self.invert(node.left)
        self.invert(node.right)

File: C#/Invert Binary Tree/Invert Binary Tree.cs
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     public int val;
 *     public TreeNode left;
 *     public TreeNode right;
 *     public TreeNode(int x) { val = x; }
 * }
 */
public class Solution {
    public TreeNode InvertTree (TreeNode root) {
        Invert (root);
        return root;
    }

    public void Invert (TreeNode node) {
        if (node == null) return;
        TreeNode tmp = node.left;
        node.left = node.right;
        node.right = tmp;
        Invert (node.left);
        Invert (node.right);
    }
}